packages/collabswarm/src/collabswarm-document.ts

```diff
@@ -664,27 +702,27 @@ export class CollabswarmDocument<
     this._fireLocalUpdateHandlers([hash]);
   }
 
-  public async pin() {
-    // Apply local change w/ CRDT provider.
-    const changes = this._crdtProvider.getHistory(this.document);
-
-    // Store changes in ipfs.
-    const newFileResult = await this.swarm.ipfsNode.add(
-      this._changesSerializer.serializeChanges(changes),
-    );
-    const hash = newFileResult.cid.toString();
-    this._hashes.add(hash);
-
-    // Send new message.
-    const updateMessage = this._createSyncMessage();
-    updateMessage.changes[hash] = changes;
-
-    if (!this.swarm.config) {
-      throw 'Can not pin a file when the node has not been initialized'!;
-    }
-    this.swarm.ipfsNode.pubsub.publish(
-      this.swarm.config.pubsubDocumentPublishPath,
-      this._syncMessageSerializer.serializeSyncMessage(updateMessage),
-    );
-  }
+  // public async pin() {
```

Comment:
Good move to comment out ::thumbsup::

Reply:
Yeah, reading through this again, I realized that the whole method doesn't work and likely never did.

Reply:
I suspect we may wind up wanting some kind of global pin UX. (I) should probably update readme to point out that data can be lost